querybook/webapp/lib/query-result/transformer.tsx

```diff
@@ -80,6 +80,16 @@ const queryResultTransformers: IColumnTransformer[] = [
         auto: false,
         transform: (v: string): React.ReactNode => v.toLocaleUpperCase(),
     },
+    {
+        key: 'preformatted',
+        name: 'Preformatted',
+        appliesToType: ['multiline'],
+        priority: 0,
+        auto: true,
+        transform: (v: string): React.ReactNode => (
+            <div className={'preformatted'}>{v}</div>
```

Comment:
is there associated css with preformatted?

also can you include a screenshot of what is before vs after?

Reply:
I updated the PR with this information. Thank you!